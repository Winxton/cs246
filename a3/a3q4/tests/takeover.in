game 5 3 stdin stdin
100 100
quit
