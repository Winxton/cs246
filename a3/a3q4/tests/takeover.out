Total is 5, max deduction is 3
Player A's move
Total is 2
Player B's move
Total is -1
Player B wins
Score is
A 0
B 1
